### PR TITLE
Rewrite browser_webconsole_bug_579412_input_focus.js

### PR DESCRIPTION
**Pull request by linclark**
Parent #292 

@bgrins this is ready for review

### CODE REVIEW

**Comment by bgrins** [comment]
Can you rebase this when you get a chance?

**Comment by linclark** [comment]
rebased

File: devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_input_focus.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* -*- indent-tabs-mode: nil; js-indent-level: 2 -*- */
+/* vim: set ft=javascript ts=2 et sw=2 tw=80: */
+/* Any copyright is dedicated to the Public Domain.
+ * http://creativecommons.org/publicdomain/zero/1.0/ */
+
+// Tests that the input field is focused when the console is opened.
+
+"use strict";
+
+const TEST_URI = "http://example.com/browser/devtools/client/webconsole/" +
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/test/mochitest/browser_webconsole_input_focus.js]: This should either load `http://example.com/browser/devtools/client/webconsole/new-console-output/test/mochitest/test-console.html` or just `"data:text/html;charset=utf-8,test"`

**Review by bgrins — APPROVED** [review]
See comment inline

**Pull request merged**